simple_client_example.py

```diff
@@ -71,9 +71,9 @@ async def get_information():
         # )
 
         # Trips
-        pp.pprint(
-            f"Trips: f{await car.get_trips(date.today() - timedelta(days=7), date.today(), full_route=True)}"  # noqa: E501
-        )
+        # pp.pprint(
```

Comment:
Did you mean to comment this out? I know its the first thing I do when working on the code as I dont want to see it. Maybe we can leave it commented out, its just there as an example.

Reply:
Yes, I felt the same way. I found myself comment out this part every time because I was plastered with GPS coordinates. 😆 
Let's comment it out for now, but leave it in as an example.